src/applications/appeals/10182/components/AddIssue.jsx

```diff
@@ -1,239 +1,16 @@
-        'button-background-color': 'white',
-      });
-      setStorage(index, 'cancel', addOrEdit === 'add' ? -1 : '');
-      goToPath(returnPath);
-    },
-    onUpdate: event => {
-      event.preventDefault();
-      recordEvent({
-        event: 'cta-button-click',
-        'button-type': 'primary',
-        'button-click-label': 'Add issue',
-        'button-background-color': 'blue',
-      });
-      setStorage(index, 'updated');
-      addOrUpdateIssue();
-    },
-  };
-
-  return (
-    <form onSubmit={handlers.onSubmit}>
-      <fieldset>
-        <legend
-          id="decision-date-description"
-          className="vads-u-font-family--serif"
-          name="addIssue"
-        >
-          <h3 className="vads-u-margin--0">{content.title[addOrEdit]}</h3>
-        </legend>
-        <VaTextInput
-          id="issue-name"
-          name="issue-name"
-          type="text"
-          label={content.name.label}
-          required
-          value={issueName}
-          onInput={handlers.onIssueNameChange}
-          onBlur={handlers.onInputBlur}
-          error={((submitted || inputDirty) && showIssueNameError) || null}
-          message-aria-describedby={content.name.hintText}
-        >
-          {content.name.hint}
-        </VaTextInput>
-
-        <br role="presentation" />
-
-        <VaMemorableDate
-          name="decision-date"
-          label={content.date.label}
-          hint={content.date.hint}
-          required
-          onDateChange={handlers.onDateChange}
-          onDateBlur={handlers.onDateBlur}
-          value={issueDate}
-          error={((submitted || dateDirty) && dateErrorMessage[0]) || null}
-          invalidMonth={isInvalid('month')}
-          invalidDay={isInvalid('day')}
-          invalidYear={isInvalid('year')}
-          aria-describedby="decision-date-description"
-        />
-        <p>
-          <va-button
-            id="cancel"
-            secondary
-            class="vads-u-width--auto"
-            onClick={handlers.onCancel}
-            text={content.button.cancel}
-          />
-          <va-button
-            id="submit"
-            class="vads-u-width--auto"
-            onClick={handlers.onUpdate}
-            text={content.button[addOrEdit]}
-          />
-        </p>
-      </fieldset>
-    </form>
-  );
-};
-
-AddIssue.propTypes = {
-  data: PropTypes.shape({}),
-  goToPath: PropTypes.func,
-  setFormData: PropTypes.func,
-  testingIndex: PropTypes.number,
-  onReviewPage: PropTypes.bool,
-};
+import { validateDate } from '../validations/date';
 
-const mapDispatchToProps = {
-  setFormData: setData,
-};
+import AddIssue from '../../shared/components/AddIssue';
 
-export default connect(
-  null,
-  mapDispatchToProps,
-)(AddIssue);
+const AddContestableIssue = props => (
+  <AddIssue
+    {...props}
+    validations={{ maxNameLength, validateDate }}
+    description={null}
+  />
+);
 
-export { AddIssue };
+export default AddContestableIssue;
```

Comment:
this is nitpicky, but can you rename these files for this `AddContestableIssue` component in all the forms to match this new name instead of being `AddIssue.jsx`? 😬 might make it easier for us to find later on

Reply:
Yeah, I can. CI keeps failing, so I'm not excited about it LOL